src/components/events/Event/index.js

```diff
@@ -64,37 +65,52 @@ module.exports = React.createClass({
     this.setState({ activeClass: '' });
   },
 
-  render: function() {
-    var baseStyles = {
-      background: this.props.backgroundColor + ' url(' + this.props.backgroundImageUrl + ')',
+  eventStyles: function() {
+    return {
+      width: this.props.width
+    };
+  },
+
+  baseStyles: function() {
+    return {
+      backgroundImage: 'url(' + this.props.backgroundImageUrl + ')',
       backgroundSize: 'cover'
     };
+  },
 
-    var blurStyles = {
+  blurStyles: function() {
+    return {
       backgroundImage: 'url(' + this.state.base64BlurredBackgroundImage + ')',
       backgroundSize: 'cover'
     };
+  },
 
-    var baseClass = "Event__base " + this.state.activeClass;
-    var date = this.props.date;
+  baseClass: function() {
+    return "Event__base " + this.state.activeClass;
+  },
+
+  render: function() {
+    var props = this.props;
+    var date = props.date;
+    var t = this.t;
 
     return (
-      <div className="Event"
+      <div className="Event" style={ this.eventStyles() }
         onTouchStart={ this.activateAndTimeout }
         onTouchCancel={ this.deactivate }
         onMouseEnter={ this.activate }
         onMouseLeave={ this.deactivate }>
-        <div className={ baseClass } style={ baseStyles }>
-          <div className="Event__blur" style={ blurStyles }></div>
+        <div className={ this.baseClass() } style={ this.baseStyles() }>
+          <div className="Event__blur" style={ this.blurStyles() }></div>
           <div className="Event__gradient"></div>
           <ul className="Event__date">
             <li>{ date.getDate() }</li>
-            <li>{ this.t('months')[date.getMonth()] }</li>
+            <li>{ t('months')[date.getMonth()] }</li>
             <li>{ date.getFullYear() }</li>
           </ul>
-          <p className="Event__name">{ this.props.name }</p>
-          <p className="Event__supporter-count">{ (this.props.supporterCount || 0) + ' ' + this.t('supportersLabel') }</p>
-          <a href={ this.props.getStartedUrl } className="Event__join-event">{ this.t('joinLabel') }</a>
+          <p className="Event__name">{ props.name }</p>
```

Comment:
I mean to work out what the link is. It's not in the data you get back, and as someone who hasn't been here for year I have no idea what it is off the top of my head. :)